readthedocs/builds/models.py

```diff
@@ -1239,6 +1238,7 @@ class VersionAutomationRule(PolymorphicModel, TimeStampedModel):
     priority = models.IntegerField(
```

Comment:
We can use `PositiveIntegerField` here instead. That way we don't need the validation for negative numbers.